bugsnag/tornado/__init__.py

```diff
@@ -38,6 +39,7 @@ def _handle_request_exception(self, exc):
         RequestHandler._handle_request_exception(self, exc)
 
     def prepare(self):
+        bugsnag.configure().runtime_versions['tornado'] = tornado.version
```

Comment:
I think this change is missing test coverage.

Reply:
It is, so the change was manually tested using the example. I've raised an internal ticket to consider adding unit tests to this integration.